gruntfile.js

```diff
@@ -56,12 +56,13 @@ module.exports = function( grunt ) {
 					'head-html-common': 'source/head-html-common.html',
 					footer: 'Copyright &copy; 2003-2017, <a href=\"http://cksource.com\" style=\"color:#085585\">CKSource</a> - Frederico Knabben. All rights reserved. | <a href=\"LICENSE.html\" style=\"color:#085585\">License</a> | Generated with <a href=\"https://github.com/senchalabs/jsduck\">JSDuck</a>.',
 					tags: 'source/customs.rb',
-					warnings: '-nodoc',
+					warnings: ['-nodoc', '-image_unused', '-link:guides/dev/code_documentation'],
```

Comment:
Rather than adding `-link:guides/dev/code_documentation` to ignore list, it's best to simply remove faulty `{@link #element}` from the guide, since it's... downcasted to text anyway.

So just replace `{@link #element}` with `element` in https://github.com/ckeditor/ckeditor-docs/blob/c92479b/guides/dev/code_documentation/README.md#L129 as it's not a crucial part from the content perspective.